substrate/client/src/serai/mod.rs

```diff
@@ -195,6 +195,19 @@ impl Serai {
     Ok(())
   }
 
+  pub async fn active_network_validators(
```

Comment:
is it best if we do this private and add a public one to ValidatorSets calls?  I get that it makes sense for it to be called through ValidatorSets calls but the implementation is on the same level as base serai client since it makes raw calls and needs helper functions from there.